### PR TITLE
Similar config to essex.gov.uk

### PR DESCRIPTION
**Pull request by keelanfh**
This PR is intended to bring across a lot of the config changes that we've made on essex.gov.uk so that there's consistency between the two sites. Specifically, this covers:

* Disabling autocomplete in Cludo search results
* Various permissions changes so that the two sites are the same (apart from group-related permissions which remain in intranet)
* Changes to layouts of views and content editing forms
* Change to linkit profile to allow linking to documents
* Footer positioned in Content menu to allow Editors to navigate to it
* Removing bulk editing content options that don't work due to conflicts with LGD's moderation stuff
* Removing the language dropdown from the media library

There are quite a few changes in here so it would be great if someone could have a quick look through the effects in dev before deploying to UAT to see that nothing breaks drastically. Everything is based on things that are already on essex.gov.uk, but it's possible that there are differences between the two environments that could cause problems.

The intranet team can then UAT to check if there are any specific undesirable changes. We did go through the changes yesterday and couldn't envisage anything here that would cause a problem for current behaviour on intranet.

### CODE REVIEW

File: config/default/core.entity_form_display.node.localgov_guides_page.default.yml [hunk 1]
```diff
@@ -3,30 +3,30 @@ langcode: en
 status: true
 dependencies:
   config:
+    - entity_browser.browser.page_components
     - field.field.node.localgov_guides_page.body
     - field.field.node.localgov_guides_page.field_content_owner
-    - field.field.node.localgov_guides_page.field_content_sme
```
Comment (siliconmeadow) [inline config/default/core.entity_form_display.node.localgov_guides_page.default.yml:8]: This is all required on intranet for Content Owner/SME functionality.

File: config/default/core.entity_form_display.node.localgov_guides_page.default.yml [hunk 2]
```diff
@@ -3,30 +3,30 @@ langcode: en
...
     group_content_ownership:
-      children:
-        - field_content_owner
-        - field_content_sme
+      children: {  }
       label: 'Content Ownership'
-      region: content
+      region: hidden
       parent_name: ''
-      weight: 14
```
Comment (siliconmeadow) [inline config/default/core.entity_form_display.node.localgov_guides_page.default.yml:29]: This is all required on intranet for Content Owner/SME functionality.

File: config/default/core.entity_form_display.node.localgov_guides_page.default.yml [hunk 3]
```diff
@@ -53,23 +53,13 @@ content:
...
-    region: content
-    settings:
-      match_operator: CONTAINS
-      match_limit: 10
-      size: 60
-      placeholder: ''
-    third_party_settings: {  }
-  field_content_sme:
-    type: entity_reference_autocomplete
-    weight: 16
```
Comment (siliconmeadow) [inline config/default/core.entity_form_display.node.localgov_guides_page.default.yml:72]: This is all required on intranet for Content Owner/SME functionality.

**Review by siliconmeadow — CHANGES_REQUESTED** [review]
This would break a lot of things if merged, I think. No, it definitely would. I just did a `drush deploy` on my local build:

```
ddev drush deploy                                                                                                                                                                                                                                                                                                       ─╯
 [notice] Database updates start.
>  [success] No pending updates.
 [success] Cache rebuild start.
>  [success] Cache rebuild complete.
 [success] Config import start.
+------------+--------------------------------------------------------------------+-----------+
| Collection | Config                                                             | Operation |
+------------+--------------------------------------------------------------------+-----------+
|            | core.entity_form_display.node.localgov_services_sublanding.default | Create    |
|            | cludo_search.settings                                              | Update    |
|            | core.menu.static_menu_link_overrides                               | Update    |
|            | linkit.linkit_profile.default                                      | Update    |
|            | core.entity_view_display.node.localgov_services_page.default       | Update    |
|            | core.entity_view_display.node.localgov_services_landing.default    | Update    |
|            | core.entity_view_display.node.localgov_news_article.default        | Update    |
|            | core.entity_view_display.node.localgov_guides_page.default         | Update    |
|            | core.entity_view_display.node.localgov_guides_overview.default     | Update    |
|            | views.view.content                                                 | Update    |
|            | views.view.media                                                   | Update    |
|            | views.view.media_library                                           | Update    |
|            | user.role.localgov_news_editor                                     | Update    |
|            | user.role.localgov_editor                                          | Update    |
|            | user.role.localgov_contributor                                     | Update    |
|            | user.role.localgov_author                                          | Update    |
|            | core.entity_form_display.node.localgov_services_page.default       | Update    |
|            | core.entity_form_display.node.localgov_services_landing.default    | Update    |
|            | core.entity_form_display.node.localgov_guides_page.default         | Update    |
|            | core.entity_form_display.node.localgov_guides_overview.default     | Update    |
|            | core.entity_form_display.node.localgov_services_sublanding.default | Delete    |
+------------+--------------------------------------------------------------------+-----------+

 Import the listed configuration changes? (yes/no) [yes]:
 > >  [error]  Drupal\Core\Config\ConfigImporterException: There were errors validating the config synchronization.
> Configuration <em class="placeholder">core.entity_form_display.node.localgov_guides_overview.default</em> depends on the <em class="placeholder">field.field.node.localgov_guides_overview.localgov_page_components</em> configuration that will not exist after import.
> Configuration <em class="placeholder">core.entity_form_display.node.localgov_guides_page.default</em> depends on the <em class="placeholder">field.field.node.localgov_guides_page.localgov_page_components</em> configuration that will not exist after import.
> Configuration <em class="placeholder">core.entity_form_display.node.localgov_services_landing.default</em> depends on the <em class="placeholder">field.field.node.localgov_services_landing.localgov_services_parent</em> configuration that will not exist after import.
> Configuration <em class="placeholder">core.entity_view_display.node.localgov_guides_overview.default</em> depends on the <em class="placeholder">field.field.node.localgov_guides_overview.localgov_page_components</em> configuration that will not exist after import.
> Configuration <em class="placeholder">core.entity_view_display.node.localgov_guides_page.default</em> depends on the <em class="placeholder">field.field.node.localgov_guides_page.localgov_page_components</em> configuration that will not exist after import.
> Configuration <em class="placeholder">core.entity_view_display.node.localgov_services_landing.default</em> depends on the <em class="placeholder">field.field.node.localgov_services_landing.localgov_services_parent</em> configuration that will not exist after import. in Drupal\Core\Config\ConfigImporter->validate() (line 788 of /var/www/html/web/core/lib/Drupal/Core/Config/ConfigImporter.php).
>
> In ConfigImportCommands.php line 324:
>
>   The import failed due to the following reasons:
>   Configuration <em class="placeholder">core.entity_form_display.node.localgov_guides_overview.default</em> depends on the <em class="placeholder">field.field.node.localgov_guides_overview.localgov_page_components</em> configuration that will not exist after import.
>   Configuration <em class="placeholder">core.entity_form_display.node.localgov_guides_page.default</em> depends on the <em class="placeholder">field.field.node.localgov_guides_page.localgov_page_components</em> configuration that will not exist after import.
>   Configuration <em class="placeholder">core.entity_form_display.node.localgov_services_landing.default</em> depends on the <em class="placeholder">field.field.node.localgov_services_landing.localgov_services_parent</em> configuration that will not exist after import.
>   Configuration <em class="placeholder">core.entity_view_display.node.localgov_guides_overview.default</em> depends on the <em class="placeholder">field.field.node.localgov_guides_overview.localgov_page_components</em> configuration that will not exist after import.
>   Configuration <em class="placeholder">core.entity_view_display.node.localgov_guides_page.default</em> depends on the <em class="placeholder">field.field.node.localgov_guides_page.localgov_page_components</em> configuration that will not exist after import.
>   Configuration <em class="placeholder">core.entity_view_display.node.localgov_services_landing.default</em> depends on the <em class="placeholder">field.field.node.localgov_services_landing.localgov_services_parent</em> configuration that will not exist after import.
>
>

In SiteProcess.php line 214:

  The command "/var/www/html/bin/drush config:import --uri=https://essex-intranet.ddev.site --root=/var/www/html/web" failed.

  Exit Code: 1(General error)

  Working directory:

  Output:
  ================
  +------------+--------------------------------------------------------------------+-----------+
  | Collection | Config                                                             | Operation |
  +------------+--------------------------------------------------------------------+-----------+
  |            | core.entity_form_display.node.localgov_services_sublanding.default | Create    |
  |            | cludo_search.settings                                              | Update    |
  |            | core.menu.static_menu_link_overrides                               | Update    |
  |            | linkit.linkit_profile.default                                      | Update    |
  |            | core.entity_view_display.node.localgov_services_page.default       | Update    |
  |            | core.entity_view_display.node.localgov_services_landing.default    | Update    |
  |            | core.entity_view_display.node.localgov_news_article.default        | Update    |
  |            | core.entity_view_display.node.localgov_guides_page.default         | Update    |
  |            | core.entity_view_display.node.localgov_guides_overview.default     | Update    |
  |            | views.view.content                                                 | Update    |
  |            | views.view.media                                                   | Update    |
  |            | views.view.media_library                                           | Update    |
  |            | user.role.localgov_news_editor                                     | Update    |
  |            | user.role.localgov_editor                                          | Update    |
  |            | user.role.localgov_contributor                                     | Update    |
  |            | user.role.localgov_author                                          | Update    |
  |            | core.entity_form_display.node.localgov_services_page.default       | Update    |
  |            | core.entity_form_display.node.localgov_services_landing.default    | Update    |
  |            | core.entity_form_display.node.localgov_guides_page.default         | Update    |
  |            | core.entity_form_display.node.localgov_guides_overview.default     | Update    |
  |            | core.entity_form_display.node.localgov_services_sublanding.default | Delete    |
  +------------+--------------------------------------------------------------------+-----------+

   Import the listed configuration changes? (yes/no) [yes]:
   >

  Error Output:
  ================
   [error]  Drupal\Core\Config\ConfigImporterException: There were errors validating the config synchronization.
  Configuration <em class="placeholder">core.entity_form_display.node.localgov_guides_overview.default</em> depends on the <em class="placeholder">field.field.node.localgov_guides_overview.localgov_page_components</em> configuration that will not exist after import.
  Configuration <em class="placeholder">core.entity_form_display.node.localgov_guides_page.default</em> depends on the <em class="placeholder">field.field.node.localgov_guides_page.localgov_page_components</em> configuration that will not exist after import.
  Configuration <em class="placeholder">core.entity_form_display.node.localgov_services_landing.default</em> depends on the <em class="placeholder">field.field.node.localgov_services_landing.localgov_services_parent</em> configuration that will not exist after import.
  Configuration <em class="placeholder">core.entity_view_display.node.localgov_guides_overview.default</em> depends on the <em class="placeholder">field.field.node.localgov_guides_overview.localgov_page_components</em> configuration that will not exist after import.
  Configuration <em class="placeholder">core.entity_view_display.node.localgov_guides_page.default</em> depends on the <em class="placeholder">field.field.node.localgov_guides_page.localgov_page_components</em> configuration that will not exist after import.
  Configuration <em class="placeholder">core.entity_view_display.node.localgov_services_landing.default</em> depends on the <em class="placeholder">field.field.node.localgov_services_landing.localgov_services_parent</em> configuration that will not exist after import. in Drupal\Core\Config\ConfigImporter->validate
  () (line 788 of /var/www/html/web/core/lib/Drupal/Core/Config/ConfigImporter.php).

  In ConfigImportCommands.php line 324:

    The import failed due to the following reasons:
    Configuration <em class="placeholder">core.entity_form_display.node.localgov_guides_overview.default</em> depends on the <em class="placeholder">field.field.node.localgov_guides_overview.localgov_page_components</em> configuration that will not exist after import.
    Configuration <em class="placeholder">core.entity_form_display.node.localgov_guides_page.default</em> depends on the <em class="placeholder">field.field.node.localgov_guides_page.localgov_page_components</em> configuration that will not exist after import.
    Configuration <em class="placeholder">core.entity_form_display.node.localgov_services_landing.default</em> depends on the <em class="placeholder">field.field.node.localgov_services_landing.localgov_services_parent</em> configuration that will not exist after import.
    Configuration <em class="placeholder">core.entity_view_display.node.localgov_guides_overview.default</em> depends on the <em class="placeholder">field.field.node.localgov_guides_overview.localgov_page_components</em> configuration that will not exist after import.
    Configuration <em class="placeholder">core.entity_view_display.node.localgov_guides_page.default</em> depends on the <em class="placeholder">field.field.node.localgov_guides_page.localgov_page_components</em> configuration that will not exist after import.
    Configuration <em class="placeholder">core.entity_view_display.node.localgov_services_landing.default</em> depends on the <em class="placeholder">field.field.node.localgov_services_landing.localgov_services_parent</em> configuration that will not exist after import.


Failed to run drush deploy: exit status 1
```

File: config/default/core.entity_form_display.node.localgov_guides_overview.default.yml [hunk 4]
```diff
@@ -3,20 +3,22 @@ langcode: en
 status: true
 dependencies:
   config:
+    - entity_browser.browser.page_components
     - field.field.node.localgov_guides_overview.body
     - field.field.node.localgov_guides_overview.field_content_owner
-    - field.field.node.localgov_guides_overview.field_content_sme
```
[the remark this answers is not in the document]
Comment (keelanfh) [reply config/default/core.entity_form_display.node.localgov_guides_overview.default.yml:8]: OK. We removed this from essex.gov.uk in https://github.com/essexcountycouncil/essex-gov-uk-drupal/pull/70 and we'd like to do the same on intranet too.

File: config/default/core.entity_form_display.node.localgov_guides_overview.default.yml [hunk 5]
```diff
@@ -98,12 +88,6 @@ content:
       rows: 5
       placeholder: ''
     third_party_settings: {  }
-  localgov_guides_list_format:
-    type: options_select
-    weight: 3
-    region: content
-    settings: {  }
-    third_party_settings: {  }
```
[the remark this answers is not in the document]
Comment (keelanfh) [reply config/default/core.entity_form_display.node.localgov_guides_overview.default.yml:106]: Selecting any of these options has no effect - it does not change the appearance of the list in the view, which was why we removed it from essex.gov.uk. We're just hiding it from the form (L193)

**Comment by siliconmeadow** [comment]
Copied the text of the body of the request for the associated spike ticket.

**Pull request closed without merging**